test/modern_bpf/test_suites/syscall_exit_suite/getuid_x.cpp

```diff
@@ -0,0 +1,39 @@
+#include "../../event_class/event_class.h"
+
+#ifdef __NR_getuid
+TEST(SyscallExit, getuidX)
+{
+	auto evt_test = get_syscall_event_test(__NR_getuid, EXIT_EVENT);
+
+	evt_test->enable_capture();
+
+	/*=============================== TRIGGER SYSCALL ===========================*/
+
+	assert_syscall_state(SYSCALL_SUCCESS, "getuid", syscall(__NR_getuid), NOT_EQUAL, -1);
+	uid_t uid = syscall(__NR_getuid);
```

Comment:
With this change:
````
uid_t uid = syscall(__NR_getuid);
assert_syscall_state(SYSCALL_SUCCESS, "getuid", uid, NOT_EQUAL, -1);
````
you could save one system call (similar approach you can find, for example, in `socket` test cases).

Reply:
Thanks! Fixed here as well as in getgid_x.cpp, geteuid_x.cpp and getegid_x.cpp.